### DISCUSSION TITLE
Generate and host a specific Tailwind CSS lightweight documentation based on our configuration

### DISCUSSION THREAD

**Discussion by lauthieb** [tailwindlabs/tailwindcss#5941] [Ideas]
Hello! 

I'm Laurent, Technical Leader of [Vitamin, Decathlon Design System](https://github.com/decathlon/vitamin-web), we are using Tailwind CSS from the beginning (june 2020) and we love it! Thanks for this!

At Decathlon, we are using a prefix in our [Tailwind configuration](https://github.com/Decathlon/vitamin-web/blob/main/packages/sources/css/config/tailwind/tailwind.config.js) called `vtmn-`, the little problem is about the documentation. For the moment we need to redirect consumers to https://tailwindcss.com and tell them to add `vtmn- ` before each generated utility CSS classes and to tell them that all the values are not correct because we extend / override some properties.

That's why I'm here to propose an idea 💡
It could be nice to generate and host our specific Tailwind CSS documentation (lightweight, without the homepage and other documentation from https://tailwindcss.com purpose only) based on our Tailwind CSS configuration, so that will show directly the right prefixes in generated utility classes and good colors / typo / spacings etc. based on our unique configuration.

Feel free if you have any question, have a great day :-)

Laurent

**Comment by lauthieb** [comment]
May I have any update please?

**Comment by NovusOpus** [comment]
You can fork [tailwindcss.com](https://github.com/tailwindlabs/tailwindcss.com) , and delete or replace the content with all the changes you need. Currently there's no License listed in that repository, So you would need to have some credit that it's based on `tailwindcss.com` source code, but even if you don't need to include any credit it's always nice to add it as courtesy and good faith.

What I don't understand though, why you summited this as an idea? Seems like you're asking for permission to host your own version of the docs, If that's the case you should contact directly to Tailwind CSS Team, you can reach them via [Twitter](https://twitter.com/tailwindcss) or [Discord](https://tailwindcss.com/discord), or also with @adamwathan(Tailwind CSS Creator).

**Reply by lauthieb** [reply]
Thanks a lot @NovusOpus for your answer!
Yes I've imagined the fork option but I thought it could be a great idea to generate this documentation from our TailwindCSS configuration file. That's why I've summited this here.. Thanks again for the contacts links!!
Have a nice day,
Laurent

**Reply by NovusOpus** [reply]
Oh I see what you mean now, There was this [Discussion](https://github.com/tailwindlabs/tailwindcss/discussions/2477) from a while ago. Maybe you can find something useful there.

**Reply by lauthieb** [reply]
Awesome, thanks!
